conf/chi.conf.example

```diff
@@ -1,24 +1,10 @@
-[storage DEFAULT]
-storage=redis
-
-[storage ldap_auth]
-expires_in=10m
-
-[storage httpd.admin]
-expires_in=1d
-
-[storage httpd.portal]
-expires_in=6h
-
-[storage redis]
-driver = Redis
-redis_class = Redis::Fast
-server = 127.0.0.1:6379
-prefix = pf
-expires_on_backend = 1
-reconnect=60
+# Look in /usr/local/pf/conf/chi.conf.defaults
+# for allowed parameters 
+# Also look in the CHI and the CHI::Driver::<Class>
+# documentation for allow parameters for cache storage
+#
 
```

Comment:
Can you add an example of modifying the cache expiration of a namspace since that would be the most common case